src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -14,23 +17,40 @@ internal unsafe partial class Number
                 exponentBits: 11,
                 maxBinaryExponent: 1023,
                 exponentBias: 1023,
-                infinityBits: 0x7FF00000_00000000
+                infinityBits: 0x7FF00000_00000000,
+                smallerPowerOfTen: -342,
```

Comment:
perhaps `minDecimalExponent` and `maxDecimalExponent` to match the existing names?

Also, is this a typo? Shouldn't this be `~4.9406564584124654 * 10^-324`?

Reply:
I've renammed both .
Constants values are good, these values came from Lemire's paper.

Reply:
Thanks, I found the relevant section in the paper and it makes sense to me. It might be good to have a comment covering this:
> Because we have that `w * 10^-343 < 2^-1074` for all `w < 2^64`, it follows that we never have to be concerned with overly small decimal exponents: when `q < −342`, then the number is assuredly zero.

Which is basically just stating that `(ulong.MaxValue * 10^-342) >= double.Epsilon` and so may be non-zero; while `10^-343 < double.Epsilon` and so will always round to zero (and more specifically, its less than `2^-1075` and so couldn't round towards epsilon either).